<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="9" ignored="0" total="35" passed="26">
  <reporter-output>
  </reporter-output>
  <suite name="Functinal_Execution" duration-ms="3763" started-at="2022-06-17T13:19:49 IST" finished-at="2022-06-17T13:19:52 IST">
    <groups>
      <group name="Regression">
        <method signature="CreateOrder.testCreateDeliveredPetOrder()[pri:3, instance:com.n26.functional.store.CreateOrder@55536d9e]" name="testCreateDeliveredPetOrder" class="com.n26.functional.store.CreateOrder"/>
        <method signature="CreateOrder.testCreateOrderWithInvalidStatus()[pri:4, instance:com.n26.functional.store.CreateOrder@55536d9e]" name="testCreateOrderWithInvalidStatus" class="com.n26.functional.store.CreateOrder"/>
        <method signature="CreateOrder.testCreatePetOrder()[pri:1, instance:com.n26.functional.store.CreateOrder@55536d9e]" name="testCreatePetOrder" class="com.n26.functional.store.CreateOrder"/>
        <method signature="CreateOrder.testCreatePlacedPetOrder()[pri:2, instance:com.n26.functional.store.CreateOrder@55536d9e]" name="testCreatePlacedPetOrder" class="com.n26.functional.store.CreateOrder"/>
        <method signature="CreatePet_POST.testCreatePet()[pri:1, instance:com.n26.functional.pets.CreatePet_POST@3e6ef8ad]" name="testCreatePet" class="com.n26.functional.pets.CreatePet_POST"/>
        <method signature="CreatePet_POST.testCreatePetWithInvalidDataType_Id()[pri:4, instance:com.n26.functional.pets.CreatePet_POST@3e6ef8ad]" name="testCreatePetWithInvalidDataType_Id" class="com.n26.functional.pets.CreatePet_POST"/>
        <method signature="CreatePet_POST.testCreatePetWithInvalidDataType_name()[pri:5, instance:com.n26.functional.pets.CreatePet_POST@3e6ef8ad]" name="testCreatePetWithInvalidDataType_name" class="com.n26.functional.pets.CreatePet_POST"/>
        <method signature="CreatePet_POST.testCreatePetWithInvalidRequestMethod()[pri:3, instance:com.n26.functional.pets.CreatePet_POST@3e6ef8ad]" name="testCreatePetWithInvalidRequestMethod" class="com.n26.functional.pets.CreatePet_POST"/>
        <method signature="CreatePet_POST.testCreatePetwWithoutMandatoryParams()[pri:2, instance:com.n26.functional.pets.CreatePet_POST@3e6ef8ad]" name="testCreatePetwWithoutMandatoryParams" class="com.n26.functional.pets.CreatePet_POST"/>
        <method signature="DeleteOrder.testDeleteOrderByInValidId()[pri:2, instance:com.n26.functional.store.DeleteOrder@747edf66]" name="testDeleteOrderByInValidId" class="com.n26.functional.store.DeleteOrder"/>
        <method signature="DeleteOrder.testDeleteOrderByValidId()[pri:1, instance:com.n26.functional.store.DeleteOrder@747edf66]" name="testDeleteOrderByValidId" class="com.n26.functional.store.DeleteOrder"/>
        <method signature="DeletePetById.testDeletePetByInvalidId()[pri:2, instance:com.n26.functional.pets.DeletePetById@62230c58]" name="testDeletePetByInvalidId" class="com.n26.functional.pets.DeletePetById"/>
        <method signature="DeletePetById.testDeletePetByValidId()[pri:1, instance:com.n26.functional.pets.DeletePetById@62230c58]" name="testDeletePetByValidId" class="com.n26.functional.pets.DeletePetById"/>
        <method signature="DeleteUser.testDeleteUserWithInValidName()[pri:2, instance:com.n26.functional.user.DeleteUser@2e55dd0c]" name="testDeleteUserWithInValidName" class="com.n26.functional.user.DeleteUser"/>
        <method signature="DeleteUser.testDeleteUserWithValidName()[pri:1, instance:com.n26.functional.user.DeleteUser@2e55dd0c]" name="testDeleteUserWithValidName" class="com.n26.functional.user.DeleteUser"/>
        <method signature="FindByTags.tesGetPetByInvalidTags()[pri:2, instance:com.n26.functional.pets.FindByTags@3738449f]" name="tesGetPetByInvalidTags" class="com.n26.functional.pets.FindByTags"/>
        <method signature="FindByTags.testGetPetByValidTag()[pri:1, instance:com.n26.functional.pets.FindByTags@3738449f]" name="testGetPetByValidTag" class="com.n26.functional.pets.FindByTags"/>
        <method signature="FindPetByStatus.testGetPetByAvailableStatus()[pri:1, instance:com.n26.functional.pets.FindPetByStatus@2cd2a21f]" name="testGetPetByAvailableStatus" class="com.n26.functional.pets.FindPetByStatus"/>
        <method signature="FindPetByStatus.testGetPetByInvalidStatus()[pri:4, instance:com.n26.functional.pets.FindPetByStatus@2cd2a21f]" name="testGetPetByInvalidStatus" class="com.n26.functional.pets.FindPetByStatus"/>
        <method signature="FindPetByStatus.testValidPetByPendingStatus()[pri:2, instance:com.n26.functional.pets.FindPetByStatus@2cd2a21f]" name="testValidPetByPendingStatus" class="com.n26.functional.pets.FindPetByStatus"/>
        <method signature="FindPetByStatus.testValidPetBySoldStatus()[pri:3, instance:com.n26.functional.pets.FindPetByStatus@2cd2a21f]" name="testValidPetBySoldStatus" class="com.n26.functional.pets.FindPetByStatus"/>
        <method signature="GetOrders.testGetOrderByInvalidId()[pri:1, instance:com.n26.functional.store.GetOrders@3d1cfad4]" name="testGetOrderByInvalidId" class="com.n26.functional.store.GetOrders"/>
        <method signature="LoginLogout.testUserLogin()[pri:1, instance:com.n26.functional.user.LoginLogout@74455848]" name="testUserLogin" class="com.n26.functional.user.LoginLogout"/>
        <method signature="LoginLogout.testUserLoginwithOnlyPassword()[pri:3, instance:com.n26.functional.user.LoginLogout@74455848]" name="testUserLoginwithOnlyPassword" class="com.n26.functional.user.LoginLogout"/>
        <method signature="LoginLogout.testUserLogout()[pri:4, instance:com.n26.functional.user.LoginLogout@74455848]" name="testUserLogout" class="com.n26.functional.user.LoginLogout"/>
        <method signature="UpdateAndGetPetById.testUpdatePetById()[pri:1, instance:com.n26.functional.pets.UpdateAndGetPetById@346d61be]" name="testUpdatePetById" class="com.n26.functional.pets.UpdateAndGetPetById"/>
        <method signature="UpdateAndGetPetById.testUpdatePetByInvaidId()[pri:1, instance:com.n26.functional.pets.UpdateAndGetPetById@346d61be]" name="testUpdatePetByInvaidId" class="com.n26.functional.pets.UpdateAndGetPetById"/>
        <method signature="UpdatePetById.testInavlidPetUpdate()[pri:2, instance:com.n26.functional.pets.UpdatePetById@69e1dd28]" name="testInavlidPetUpdate" class="com.n26.functional.pets.UpdatePetById"/>
        <method signature="UpdatePetById.testValidPetUpdateWithNameStatus()[pri:1, instance:com.n26.functional.pets.UpdatePetById@69e1dd28]" name="testValidPetUpdateWithNameStatus" class="com.n26.functional.pets.UpdatePetById"/>
        <method signature="UpdateUser.testUpdateUserWIthInvalidUserName()[pri:2, instance:com.n26.functional.user.UpdateUser@e7edb54]" name="testUpdateUserWIthInvalidUserName" class="com.n26.functional.user.UpdateUser"/>
        <method signature="UpdateUser.testUpdateUserWithValidName()[pri:1, instance:com.n26.functional.user.UpdateUser@e7edb54]" name="testUpdateUserWithValidName" class="com.n26.functional.user.UpdateUser"/>
        <method signature="UplodPetImage.testUploadingPetImage()[pri:1, instance:com.n26.functional.pets.UplodPetImage@11bd0f3b]" name="testUploadingPetImage" class="com.n26.functional.pets.UplodPetImage"/>
        <method signature="UserCreation.testMultipleUserCreation()[pri:2, instance:com.n26.functional.user.UserCreation@378542de]" name="testMultipleUserCreation" class="com.n26.functional.user.UserCreation"/>
        <method signature="UserCreation.testSingleUserCreation()[pri:1, instance:com.n26.functional.user.UserCreation@378542de]" name="testSingleUserCreation" class="com.n26.functional.user.UserCreation"/>
      </group> <!-- Regression -->
      <group name="Smoke">
        <method signature="CreateOrder.testCreatePlacedPetOrder()[pri:2, instance:com.n26.functional.store.CreateOrder@55536d9e]" name="testCreatePlacedPetOrder" class="com.n26.functional.store.CreateOrder"/>
        <method signature="DeleteOrder.testDeleteOrderByValidId()[pri:1, instance:com.n26.functional.store.DeleteOrder@747edf66]" name="testDeleteOrderByValidId" class="com.n26.functional.store.DeleteOrder"/>
        <method signature="DeletePetById.testDeletePetByValidId()[pri:1, instance:com.n26.functional.pets.DeletePetById@62230c58]" name="testDeletePetByValidId" class="com.n26.functional.pets.DeletePetById"/>
        <method signature="FindByTags.testGetPetByValidTag()[pri:1, instance:com.n26.functional.pets.FindByTags@3738449f]" name="testGetPetByValidTag" class="com.n26.functional.pets.FindByTags"/>
        <method signature="FindPetByStatus.testGetPetByAvailableStatus()[pri:1, instance:com.n26.functional.pets.FindPetByStatus@2cd2a21f]" name="testGetPetByAvailableStatus" class="com.n26.functional.pets.FindPetByStatus"/>
        <method signature="LoginLogout.testUserLogin()[pri:1, instance:com.n26.functional.user.LoginLogout@74455848]" name="testUserLogin" class="com.n26.functional.user.LoginLogout"/>
        <method signature="LoginLogout.testUserLogout()[pri:4, instance:com.n26.functional.user.LoginLogout@74455848]" name="testUserLogout" class="com.n26.functional.user.LoginLogout"/>
        <method signature="UpdateAndGetPetById.testUpdatePetById()[pri:1, instance:com.n26.functional.pets.UpdateAndGetPetById@346d61be]" name="testUpdatePetById" class="com.n26.functional.pets.UpdateAndGetPetById"/>
        <method signature="UpdateUser.testUpdateUserWithValidName()[pri:1, instance:com.n26.functional.user.UpdateUser@e7edb54]" name="testUpdateUserWithValidName" class="com.n26.functional.user.UpdateUser"/>
      </group> <!-- Smoke -->
    </groups>
    <test name="Sample test1" duration-ms="3763" started-at="2022-06-17T13:19:49 IST" finished-at="2022-06-17T13:19:52 IST">
      <class name="com.n26.functional.user.DeleteUser">
        <test-method status="PASS" signature="scriptStart()[pri:0, instance:com.n26.functional.user.DeleteUser@2e55dd0c]" name="scriptStart" is-config="true" duration-ms="1" started-at="2022-06-17T13:19:52 IST" finished-at="2022-06-17T13:19:52 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- scriptStart -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.n26.functional.user.DeleteUser@2e55dd0c]" name="setUp" is-config="true" duration-ms="40" started-at="2022-06-17T13:19:52 IST" finished-at="2022-06-17T13:19:52 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="testDeleteUserWithValidName()[pri:1, instance:com.n26.functional.user.DeleteUser@2e55dd0c]" name="testDeleteUserWithValidName" duration-ms="32" started-at="2022-06-17T13:19:52 IST" finished-at="2022-06-17T13:19:52 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testDeleteUserWithValidName -->
        <test-method status="FAIL" signature="testDeleteUserWithInValidName()[pri:2, instance:com.n26.functional.user.DeleteUser@2e55dd0c]" name="testDeleteUserWithInValidName" duration-ms="7" started-at="2022-06-17T13:19:52 IST" finished-at="2022-06-17T13:19:52 IST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:751)
at org.testng.Assert.assertEquals(Assert.java:761)
at com.n26.functional.user.DeleteUser.testDeleteUserWithInValidName(DeleteUser.java:79)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:324)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:74)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:123)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:456)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:169)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:595)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:581)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testDeleteUserWithInValidName -->
      </class> <!-- com.n26.functional.user.DeleteUser -->
      <class name="com.n26.functional.pets.CreatePet_POST">
        <test-method status="PASS" signature="scriptStart()[pri:0, instance:com.n26.functional.pets.CreatePet_POST@3e6ef8ad]" name="scriptStart" is-config="true" duration-ms="133" started-at="2022-06-17T13:19:49 IST" finished-at="2022-06-17T13:19:49 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- scriptStart -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.n26.functional.pets.CreatePet_POST@3e6ef8ad]" name="setUp" is-config="true" duration-ms="420" started-at="2022-06-17T13:19:49 IST" finished-at="2022-06-17T13:19:49 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="testCreatePet()[pri:1, instance:com.n26.functional.pets.CreatePet_POST@3e6ef8ad]" name="testCreatePet" duration-ms="987" started-at="2022-06-17T13:19:49 IST" finished-at="2022-06-17T13:19:50 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCreatePet -->
        <test-method status="FAIL" signature="testCreatePetwWithoutMandatoryParams()[pri:2, instance:com.n26.functional.pets.CreatePet_POST@3e6ef8ad]" name="testCreatePetwWithoutMandatoryParams" duration-ms="409" started-at="2022-06-17T13:19:50 IST" finished-at="2022-06-17T13:19:51 IST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:751)
at org.testng.Assert.assertEquals(Assert.java:761)
at com.n26.functional.pets.CreatePet_POST.testCreatePetwWithoutMandatoryParams(CreatePet_POST.java:64)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:324)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:74)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:123)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:456)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:169)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:595)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:581)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCreatePetwWithoutMandatoryParams -->
        <test-method status="PASS" signature="testCreatePetWithInvalidRequestMethod()[pri:3, instance:com.n26.functional.pets.CreatePet_POST@3e6ef8ad]" name="testCreatePetWithInvalidRequestMethod" duration-ms="72" started-at="2022-06-17T13:19:51 IST" finished-at="2022-06-17T13:19:51 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCreatePetWithInvalidRequestMethod -->
        <test-method status="PASS" signature="testCreatePetWithInvalidDataType_Id()[pri:4, instance:com.n26.functional.pets.CreatePet_POST@3e6ef8ad]" name="testCreatePetWithInvalidDataType_Id" duration-ms="103" started-at="2022-06-17T13:19:51 IST" finished-at="2022-06-17T13:19:51 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCreatePetWithInvalidDataType_Id -->
        <test-method status="FAIL" signature="testCreatePetWithInvalidDataType_name()[pri:5, instance:com.n26.functional.pets.CreatePet_POST@3e6ef8ad]" name="testCreatePetWithInvalidDataType_name" duration-ms="14" started-at="2022-06-17T13:19:51 IST" finished-at="2022-06-17T13:19:51 IST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:751)
at org.testng.Assert.assertEquals(Assert.java:761)
at com.n26.functional.pets.CreatePet_POST.testCreatePetWithInvalidDataType_name(CreatePet_POST.java:101)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:324)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:74)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:123)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:456)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:169)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:595)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:581)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCreatePetWithInvalidDataType_name -->
      </class> <!-- com.n26.functional.pets.CreatePet_POST -->
      <class name="com.n26.functional.store.CreateOrder">
        <test-method status="PASS" signature="scriptStart()[pri:0, instance:com.n26.functional.store.CreateOrder@55536d9e]" name="scriptStart" is-config="true" duration-ms="1" started-at="2022-06-17T13:19:51 IST" finished-at="2022-06-17T13:19:51 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- scriptStart -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.n26.functional.store.CreateOrder@55536d9e]" name="setUp" is-config="true" duration-ms="1" started-at="2022-06-17T13:19:51 IST" finished-at="2022-06-17T13:19:51 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="testCreatePetOrder()[pri:1, instance:com.n26.functional.store.CreateOrder@55536d9e]" name="testCreatePetOrder" duration-ms="35" started-at="2022-06-17T13:19:51 IST" finished-at="2022-06-17T13:19:51 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCreatePetOrder -->
        <test-method status="PASS" signature="testCreatePlacedPetOrder()[pri:2, instance:com.n26.functional.store.CreateOrder@55536d9e]" name="testCreatePlacedPetOrder" duration-ms="71" started-at="2022-06-17T13:19:51 IST" finished-at="2022-06-17T13:19:51 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCreatePlacedPetOrder -->
        <test-method status="PASS" signature="testCreateDeliveredPetOrder()[pri:3, instance:com.n26.functional.store.CreateOrder@55536d9e]" name="testCreateDeliveredPetOrder" duration-ms="84" started-at="2022-06-17T13:19:51 IST" finished-at="2022-06-17T13:19:51 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCreateDeliveredPetOrder -->
        <test-method status="FAIL" signature="testCreateOrderWithInvalidStatus()[pri:4, instance:com.n26.functional.store.CreateOrder@55536d9e]" name="testCreateOrderWithInvalidStatus" duration-ms="11" started-at="2022-06-17T13:19:51 IST" finished-at="2022-06-17T13:19:51 IST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:751)
at org.testng.Assert.assertEquals(Assert.java:761)
at com.n26.functional.store.CreateOrder.testCreateOrderWithInvalidStatus(CreateOrder.java:114)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:324)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:74)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:123)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:456)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:169)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:595)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:581)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCreateOrderWithInvalidStatus -->
      </class> <!-- com.n26.functional.store.CreateOrder -->
      <class name="com.n26.functional.pets.FindPetByStatus">
        <test-method status="PASS" signature="scriptStart()[pri:0, instance:com.n26.functional.pets.FindPetByStatus@2cd2a21f]" name="scriptStart" is-config="true" duration-ms="0" started-at="2022-06-17T13:19:51 IST" finished-at="2022-06-17T13:19:51 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- scriptStart -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.n26.functional.pets.FindPetByStatus@2cd2a21f]" name="setUp" is-config="true" duration-ms="1" started-at="2022-06-17T13:19:51 IST" finished-at="2022-06-17T13:19:51 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="testGetPetByAvailableStatus()[pri:1, instance:com.n26.functional.pets.FindPetByStatus@2cd2a21f]" name="testGetPetByAvailableStatus" duration-ms="50" started-at="2022-06-17T13:19:51 IST" finished-at="2022-06-17T13:19:51 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testGetPetByAvailableStatus -->
        <test-method status="PASS" signature="testValidPetByPendingStatus()[pri:2, instance:com.n26.functional.pets.FindPetByStatus@2cd2a21f]" name="testValidPetByPendingStatus" duration-ms="31" started-at="2022-06-17T13:19:51 IST" finished-at="2022-06-17T13:19:51 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testValidPetByPendingStatus -->
        <test-method status="PASS" signature="testValidPetBySoldStatus()[pri:3, instance:com.n26.functional.pets.FindPetByStatus@2cd2a21f]" name="testValidPetBySoldStatus" duration-ms="28" started-at="2022-06-17T13:19:51 IST" finished-at="2022-06-17T13:19:51 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testValidPetBySoldStatus -->
        <test-method status="PASS" signature="testGetPetByInvalidStatus()[pri:4, instance:com.n26.functional.pets.FindPetByStatus@2cd2a21f]" name="testGetPetByInvalidStatus" duration-ms="17" started-at="2022-06-17T13:19:51 IST" finished-at="2022-06-17T13:19:52 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testGetPetByInvalidStatus -->
      </class> <!-- com.n26.functional.pets.FindPetByStatus -->
      <class name="com.n26.functional.user.LoginLogout">
        <test-method status="PASS" signature="scriptStart()[pri:0, instance:com.n26.functional.user.LoginLogout@74455848]" name="scriptStart" is-config="true" duration-ms="1" started-at="2022-06-17T13:19:52 IST" finished-at="2022-06-17T13:19:52 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- scriptStart -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.n26.functional.user.LoginLogout@74455848]" name="setUp" is-config="true" duration-ms="1" started-at="2022-06-17T13:19:52 IST" finished-at="2022-06-17T13:19:52 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="testUserLogin()[pri:1, instance:com.n26.functional.user.LoginLogout@74455848]" name="testUserLogin" duration-ms="38" started-at="2022-06-17T13:19:52 IST" finished-at="2022-06-17T13:19:52 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testUserLogin -->
        <test-method status="FAIL" signature="testUserLoginwithOnlyUserName()[pri:2, instance:com.n26.functional.user.LoginLogout@74455848]" name="testUserLoginwithOnlyUserName" duration-ms="56" started-at="2022-06-17T13:19:52 IST" finished-at="2022-06-17T13:19:52 IST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:751)
at org.testng.Assert.assertEquals(Assert.java:761)
at com.n26.functional.user.LoginLogout.testUserLoginwithOnlyUserName(LoginLogout.java:59)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:324)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:74)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:123)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:456)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:169)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:595)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:581)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testUserLoginwithOnlyUserName -->
        <test-method status="FAIL" signature="testUserLoginwithOnlyPassword()[pri:3, instance:com.n26.functional.user.LoginLogout@74455848]" name="testUserLoginwithOnlyPassword" duration-ms="33" started-at="2022-06-17T13:19:52 IST" finished-at="2022-06-17T13:19:52 IST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:751)
at org.testng.Assert.assertEquals(Assert.java:761)
at com.n26.functional.user.LoginLogout.testUserLoginwithOnlyPassword(LoginLogout.java:72)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:324)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:74)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:123)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:456)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:169)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:595)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:581)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testUserLoginwithOnlyPassword -->
        <test-method status="PASS" signature="testUserLogout()[pri:4, instance:com.n26.functional.user.LoginLogout@74455848]" name="testUserLogout" duration-ms="12" started-at="2022-06-17T13:19:52 IST" finished-at="2022-06-17T13:19:52 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testUserLogout -->
      </class> <!-- com.n26.functional.user.LoginLogout -->
      <class name="com.n26.functional.store.DeleteOrder">
        <test-method status="PASS" signature="scriptStart()[pri:0, instance:com.n26.functional.store.DeleteOrder@747edf66]" name="scriptStart" is-config="true" duration-ms="1" started-at="2022-06-17T13:19:51 IST" finished-at="2022-06-17T13:19:51 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- scriptStart -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.n26.functional.store.DeleteOrder@747edf66]" name="setUp" is-config="true" duration-ms="2" started-at="2022-06-17T13:19:51 IST" finished-at="2022-06-17T13:19:51 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="testDeleteOrderByValidId()[pri:1, instance:com.n26.functional.store.DeleteOrder@747edf66]" name="testDeleteOrderByValidId" duration-ms="73" started-at="2022-06-17T13:19:51 IST" finished-at="2022-06-17T13:19:51 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testDeleteOrderByValidId -->
        <test-method status="FAIL" signature="testDeleteOrderByInValidId()[pri:2, instance:com.n26.functional.store.DeleteOrder@747edf66]" name="testDeleteOrderByInValidId" duration-ms="30" started-at="2022-06-17T13:19:51 IST" finished-at="2022-06-17T13:19:51 IST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:751)
at org.testng.Assert.assertEquals(Assert.java:761)
at com.n26.functional.store.DeleteOrder.testDeleteOrderByInValidId(DeleteOrder.java:84)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:324)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:74)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:123)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:456)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:169)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:595)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:581)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testDeleteOrderByInValidId -->
      </class> <!-- com.n26.functional.store.DeleteOrder -->
      <class name="com.n26.functional.pets.UplodPetImage">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.n26.functional.pets.UplodPetImage@11bd0f3b]" name="setUp" is-config="true" duration-ms="1" started-at="2022-06-17T13:19:52 IST" finished-at="2022-06-17T13:19:52 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="scriptStart()[pri:0, instance:com.n26.functional.pets.UplodPetImage@11bd0f3b]" name="scriptStart" is-config="true" duration-ms="0" started-at="2022-06-17T13:19:52 IST" finished-at="2022-06-17T13:19:52 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- scriptStart -->
        <test-method status="FAIL" signature="testUploadingPetImage()[pri:1, instance:com.n26.functional.pets.UplodPetImage@11bd0f3b]" name="testUploadingPetImage" duration-ms="31" started-at="2022-06-17T13:19:52 IST" finished-at="2022-06-17T13:19:52 IST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:751)
at org.testng.Assert.assertEquals(Assert.java:761)
at com.n26.functional.pets.UplodPetImage.testUploadingPetImage(UplodPetImage.java:58)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:324)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:74)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:123)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:456)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:169)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:595)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:581)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testUploadingPetImage -->
      </class> <!-- com.n26.functional.pets.UplodPetImage -->
      <class name="com.n26.functional.pets.UpdateAndGetPetById">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.n26.functional.pets.UpdateAndGetPetById@346d61be]" name="setUp" is-config="true" duration-ms="1" started-at="2022-06-17T13:19:51 IST" finished-at="2022-06-17T13:19:51 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="scriptStart()[pri:0, instance:com.n26.functional.pets.UpdateAndGetPetById@346d61be]" name="scriptStart" is-config="true" duration-ms="0" started-at="2022-06-17T13:19:51 IST" finished-at="2022-06-17T13:19:51 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- scriptStart -->
        <test-method status="PASS" signature="testUpdatePetById()[pri:1, instance:com.n26.functional.pets.UpdateAndGetPetById@346d61be]" name="testUpdatePetById" duration-ms="120" started-at="2022-06-17T13:19:51 IST" finished-at="2022-06-17T13:19:51 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testUpdatePetById -->
        <test-method status="PASS" signature="testUpdatePetByInvaidId()[pri:1, instance:com.n26.functional.pets.UpdateAndGetPetById@346d61be]" name="testUpdatePetByInvaidId" duration-ms="59" started-at="2022-06-17T13:19:51 IST" finished-at="2022-06-17T13:19:51 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testUpdatePetByInvaidId -->
      </class> <!-- com.n26.functional.pets.UpdateAndGetPetById -->
      <class name="com.n26.functional.pets.DeletePetById">
        <test-method status="PASS" signature="scriptStart()[pri:0, instance:com.n26.functional.pets.DeletePetById@62230c58]" name="scriptStart" is-config="true" duration-ms="1" started-at="2022-06-17T13:19:51 IST" finished-at="2022-06-17T13:19:51 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- scriptStart -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.n26.functional.pets.DeletePetById@62230c58]" name="setUp" is-config="true" duration-ms="1" started-at="2022-06-17T13:19:51 IST" finished-at="2022-06-17T13:19:51 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="testDeletePetByValidId()[pri:1, instance:com.n26.functional.pets.DeletePetById@62230c58]" name="testDeletePetByValidId" duration-ms="30" started-at="2022-06-17T13:19:51 IST" finished-at="2022-06-17T13:19:51 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testDeletePetByValidId -->
        <test-method status="FAIL" signature="testDeletePetByInvalidId()[pri:2, instance:com.n26.functional.pets.DeletePetById@62230c58]" name="testDeletePetByInvalidId" duration-ms="7" started-at="2022-06-17T13:19:51 IST" finished-at="2022-06-17T13:19:51 IST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:751)
at org.testng.Assert.assertEquals(Assert.java:761)
at com.n26.functional.pets.DeletePetById.testDeletePetByInvalidId(DeletePetById.java:71)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:324)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:74)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:123)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:456)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:169)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:595)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:581)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testDeletePetByInvalidId -->
      </class> <!-- com.n26.functional.pets.DeletePetById -->
      <class name="com.n26.functional.pets.FindByTags">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.n26.functional.pets.FindByTags@3738449f]" name="setUp" is-config="true" duration-ms="1" started-at="2022-06-17T13:19:52 IST" finished-at="2022-06-17T13:19:52 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="scriptStart()[pri:0, instance:com.n26.functional.pets.FindByTags@3738449f]" name="scriptStart" is-config="true" duration-ms="0" started-at="2022-06-17T13:19:52 IST" finished-at="2022-06-17T13:19:52 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- scriptStart -->
        <test-method status="PASS" signature="testGetPetByValidTag()[pri:1, instance:com.n26.functional.pets.FindByTags@3738449f]" name="testGetPetByValidTag" duration-ms="91" started-at="2022-06-17T13:19:52 IST" finished-at="2022-06-17T13:19:52 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testGetPetByValidTag -->
        <test-method status="PASS" signature="tesGetPetByInvalidTags()[pri:2, instance:com.n26.functional.pets.FindByTags@3738449f]" name="tesGetPetByInvalidTags" duration-ms="51" started-at="2022-06-17T13:19:52 IST" finished-at="2022-06-17T13:19:52 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tesGetPetByInvalidTags -->
      </class> <!-- com.n26.functional.pets.FindByTags -->
      <class name="com.n26.functional.user.UpdateUser">
        <test-method status="PASS" signature="scriptStart()[pri:0, instance:com.n26.functional.user.UpdateUser@e7edb54]" name="scriptStart" is-config="true" duration-ms="1" started-at="2022-06-17T13:19:52 IST" finished-at="2022-06-17T13:19:52 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- scriptStart -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.n26.functional.user.UpdateUser@e7edb54]" name="setUp" is-config="true" duration-ms="31" started-at="2022-06-17T13:19:52 IST" finished-at="2022-06-17T13:19:52 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="testUpdateUserWithValidName()[pri:1, instance:com.n26.functional.user.UpdateUser@e7edb54]" name="testUpdateUserWithValidName" duration-ms="95" started-at="2022-06-17T13:19:52 IST" finished-at="2022-06-17T13:19:52 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testUpdateUserWithValidName -->
        <test-method status="PASS" signature="testUpdateUserWIthInvalidUserName()[pri:2, instance:com.n26.functional.user.UpdateUser@e7edb54]" name="testUpdateUserWIthInvalidUserName" duration-ms="8" started-at="2022-06-17T13:19:52 IST" finished-at="2022-06-17T13:19:52 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testUpdateUserWIthInvalidUserName -->
      </class> <!-- com.n26.functional.user.UpdateUser -->
      <class name="com.n26.functional.store.GetOrders">
        <test-method status="PASS" signature="scriptStart()[pri:0, instance:com.n26.functional.store.GetOrders@3d1cfad4]" name="scriptStart" is-config="true" duration-ms="1" started-at="2022-06-17T13:19:51 IST" finished-at="2022-06-17T13:19:51 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- scriptStart -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.n26.functional.store.GetOrders@3d1cfad4]" name="setUp" is-config="true" duration-ms="1" started-at="2022-06-17T13:19:51 IST" finished-at="2022-06-17T13:19:51 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="testGetOrderByInvalidId()[pri:1, instance:com.n26.functional.store.GetOrders@3d1cfad4]" name="testGetOrderByInvalidId" duration-ms="29" started-at="2022-06-17T13:19:51 IST" finished-at="2022-06-17T13:19:51 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testGetOrderByInvalidId -->
      </class> <!-- com.n26.functional.store.GetOrders -->
      <class name="com.n26.functional.user.UserCreation">
        <test-method status="PASS" signature="scriptStart()[pri:0, instance:com.n26.functional.user.UserCreation@378542de]" name="scriptStart" is-config="true" duration-ms="1" started-at="2022-06-17T13:19:52 IST" finished-at="2022-06-17T13:19:52 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- scriptStart -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.n26.functional.user.UserCreation@378542de]" name="setUp" is-config="true" duration-ms="32" started-at="2022-06-17T13:19:52 IST" finished-at="2022-06-17T13:19:52 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="testSingleUserCreation()[pri:1, instance:com.n26.functional.user.UserCreation@378542de]" name="testSingleUserCreation" duration-ms="39" started-at="2022-06-17T13:19:52 IST" finished-at="2022-06-17T13:19:52 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSingleUserCreation -->
        <test-method status="PASS" signature="testMultipleUserCreation()[pri:2, instance:com.n26.functional.user.UserCreation@378542de]" name="testMultipleUserCreation" duration-ms="56" started-at="2022-06-17T13:19:52 IST" finished-at="2022-06-17T13:19:52 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testMultipleUserCreation -->
      </class> <!-- com.n26.functional.user.UserCreation -->
      <class name="com.n26.functional.pets.UpdatePetById">
        <test-method status="PASS" signature="scriptStart()[pri:0, instance:com.n26.functional.pets.UpdatePetById@69e1dd28]" name="scriptStart" is-config="true" duration-ms="0" started-at="2022-06-17T13:19:52 IST" finished-at="2022-06-17T13:19:52 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- scriptStart -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.n26.functional.pets.UpdatePetById@69e1dd28]" name="setUp" is-config="true" duration-ms="0" started-at="2022-06-17T13:19:52 IST" finished-at="2022-06-17T13:19:52 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="testValidPetUpdateWithNameStatus()[pri:1, instance:com.n26.functional.pets.UpdatePetById@69e1dd28]" name="testValidPetUpdateWithNameStatus" duration-ms="192" started-at="2022-06-17T13:19:52 IST" finished-at="2022-06-17T13:19:52 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testValidPetUpdateWithNameStatus -->
        <test-method status="PASS" signature="testInavlidPetUpdate()[pri:2, instance:com.n26.functional.pets.UpdatePetById@69e1dd28]" name="testInavlidPetUpdate" duration-ms="10" started-at="2022-06-17T13:19:52 IST" finished-at="2022-06-17T13:19:52 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testInavlidPetUpdate -->
      </class> <!-- com.n26.functional.pets.UpdatePetById -->
    </test> <!-- Sample test1 -->
  </suite> <!-- Functinal_Execution -->
</testng-results>
